src/Sylius/Behat/Service/Provider/RequestApiPathPrefixProvider.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Why there is a need to overwrite that service in behat?

Reply:
Because sometimes in our tests we use `IriConverter` before request is called, To match proper path prefix we need request.